File: tests/expected/classes.cpp
#include <cassert>   // NOLINT(build/include_order)
#include <cstdint>   // NOLINT(build/include_order)
#include <iostream>  // NOLINT(build/include_order)
class Foo {
 public:
  inline int bar() { return this->baz(); }

  inline int baz() { return 10; }
};

int main(int argc, char** argv) {
  Foo f = Foo();
  auto b = f.bar();
  std::cout << b;
  std::cout << std::endl;
  assert(b == 10);
}

File: tests/expected/classes.dart
// @dart=3.4
import 'package:sprintf/sprintf.dart';

class Foo {
  int bar() {
    return baz();
  }

  int baz() {
    return 10;
  }
}

main(List<String> argv) {
  final Foo f = Foo();
  final b = f.bar();
  print(sprintf("%s", [b]));
  assert(b == 10);
}
